tasks/prime_hhg_workflow.py

```diff
@@ -27,7 +28,10 @@ class PrimeWorkflowTasks(PrimeTasks, SupportTasks):
 
     # current_move is the move the whole task will be acting on
     # all updates to the move and nested objects will be stored to this mto
-    current_move = None
+    # current_move = None
+
+    # The id for the move we are fetching
+    current_move_id = None
```

Comment:
The workaround solution pulls the move id from the original move rather than a part of the move as we did originally. It addresses the issue of pulling stale moves that was causing errors in certain endpoints and preventing a payment request from being created.

The original code was left commented in case we decide to reimplement that solution for certain endpoints.